README.md

```diff
@@ -1,3 +1,27 @@
+# The Data Shed usage
+
+This repo (and the dependecy pywb) has been forked to make changes to allow uploads of archives and indexes to S3. You will need to set up aws credentials locally. Note the AWS_PROFILE and S3BUCKET environment variables are set.
```

Comment:
_"…the AWS_PROFILE and S3BUCKET environment…"_—should reference `AWS_S3_BUCKET`.